fvm/env.go

```diff
@@ -333,8 +333,7 @@ func (e *hostEnv) VerifySignature(
 	)
 
 	if err != nil {
-		// TODO: improve error passing https://github.com/onflow/cadence/issues/202
-		panic(err)
+		return false, nil
```

Comment:
just want to make sure this is intended. We might return `false, err`

Reply:
good catch, that was an oversight.